.github/workflows/update-canary.yml

```
@@ -23,7 +23,7 @@ jobs:
           node-version: lts/*
 
       - name: Install dependencies
-        run: npm install -g node-core-utils@latest
+        run: npm install -g @node-core/utils
 
       - name: Cache V8 clone
         uses: actions/cache@v3
```

We might as well update the other actions, wdyt?
```suggestion
        uses: actions/cache@v4
```

---

Nevermind, https://github.com/actions/cache/releases doesn't show a v4, I must have misremembered.